website_sale_recently_viewed_products/README.rst

```diff
@@ -0,0 +1,50 @@
+========================
```

Comment:
@pedrobaeza while I'm already doing this, I'm for putting the license banner (and, eventually, other banners) just under the module name / title instead of above, what do you think?